build.sbt

```diff
@@ -880,6 +949,8 @@ lazy val `project-manager` = (project in file("lib/scala/project-manager"))
   .dependsOn(`polyglot-api`)
   .dependsOn(`runtime-version-manager`)
   .dependsOn(`library-manager`)
+  .dependsOn(`logging-utils-akka`)
```

Comment:
Can we ensure this _implementation modules_ do not appear on compilation classpath of modules that are only supposed to use the logging? I'd like to avoid anyone using `ch.qos.logback` directly.

Reply:
In `project-manager`? Tricky. I will see if `sbt` can support that request.

Reply:
done